syntax = "proto3";

package pingpong;

service PingPong {
  rpc Ping (PingRequest) returns (PongResponse);
}

message PingRequest {}
message PongResponse {
  string message = 1;
}
